Scripts/developer_scripts/cgal_create_release_with_cmake.cmake

```diff
@@ -0,0 +1,110 @@
+#option :
+# DESTINATION the path where the release is create, default is /tmp
+# PUBLIC=[ON/OFF] indicates if a public release should be built
+# INTERNAL=[ON/OFF] indicates if an internal release should be built
+# VERBOSE=[ON/OFF] makes the script more verbose
+
+if (NOT EXISTS ${CMAKE_BINARY_DIR}/Installation/include/CGAL/version.h)
+  message(FATAL_ERROR "Cannot find Installation/include/CGAL/version.h. Make sure you are at the root of a CGAL branch")
+endif()
+
+file(READ "${CMAKE_BINARY_DIR}/Installation/include/CGAL/version.h" version_file_content)
+string(REGEX MATCH "define CGAL_VERSION (.*)\n#define CGAL_VERSION_NR" CGAL_VERSION_FOUND "${version_file_content}")
+
+if (CGAL_VERSION_FOUND)
+  set(CGAL_VERSION "${CMAKE_MATCH_1}")
+  set (GITHUB_PREFIX "https://github.com/CGAL/cgal/blob/releases/CGAL-${CGAL_VERSION}")
+else()
```

Comment:
That URL will be almost always wrong. Instead you should use the real hash,`${CGAL_GIT_HASH}`, after having included `Installation/cmake/modules/CGAL_SCM.cmake`.

Reply:
The URL will indeed be wrong for internal releases but is perfectly fine for official public releases.
Using the hash is not a solution for internal releases since the hash will no longer be valid after a week (IIRC we don't have tags for Ic/I releases).